tests/ui/solver-attribute/not-found/expected

```diff
@@ -0,0 +1 @@
+Error: Specified solver "non_existing_solver" not found in path
```

Comment:
I was wondering if there is a better message here. It would be odd if the user just misspelled "kissat" and they get this error.

Reply:
I couldn't think of a better one. Any suggestions?